samples/README.md

```diff
@@ -25,7 +25,7 @@ TIDES data packages must include a [`datapackage.json`](https://specs.frictionle
 | **Field** | **Description** | **Required** |
 | --------- | --------------- | ------------ |
 | `title` | A human-readable title. | Required |
-| `name` | Identifier string as a URL-friendly slug. | Required |
+| `name` | Short identifier string as a [URL-friendly slug](https://en.wikipedia.org/wiki/Clean_URL#Slug). | Recommended |
```

Comment:
Same question as line 44 below, plus these descriptions should match

Reply:
Made more consistent.